week-2/implement/bmi.js

```diff
@@ -13,3 +13,13 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+function bmi(weight, height) {
+  squaredHeight = height * height;
```

Comment:
Remember if you're defining variables inside the function `bmi`, you still need to use `const` or `let` declarations.